tests/pytest_tests/unit_tests/test_ssl.py

```diff
@@ -84,6 +85,7 @@ def disable_ssl_context():
         reset()
 
 
+@pytest.mark.xfail(sys.version_info[:2] == (3, 7), reason="fails on python version 3.7")
```

Comment:
Maybe skipif, if know that these tests are going to fail, why waste time on running?